src/librustc/session/config.rs

```diff
@@ -1045,6 +1045,8 @@ options! {CodegenOptions, CodegenSetter, basic_codegen_options,
 options! {DebuggingOptions, DebuggingSetter, basic_debugging_options,
          build_debugging_options, "Z", "debugging",
          DB_OPTIONS, db_type_desc, dbsetters,
+    trans: Option<String> = (None, parse_opt_string, [TRACKED],
+        "the backend to use"),
```

Comment:
Can you rename the option to `codegen-backend`?

Reply:
will do it